spacelift/internal/structs/version.go

```diff
@@ -0,0 +1,57 @@
+package structs
+
+type Version struct {
+	ID     string `graphql:"id"`
+	Commit struct {
+		AuthorLogin string `graphql:"authorLogin"`
+		AuthorName  string `graphql:"authorName"`
+		Hash        string `graphql:"hash"`
+		Message     string `graphql:"message"`
+		Timestamp   int    `graphql:"timestamp"`
+		URL         string `graphql:"url"`
+	} `graphql:"commit"`
+	DownloadLink *string `graphql:"downloadLink"`
+	Notes        *string `graphql:"notes"`
+	Number       string  `graphql:"number"`
+	Metadata     struct {
```

Comment:
I think I'd omit unused fields.

Reply:
I've removed this entirely, just left a small struct I defined in the resource itself. Less trouble this way imo.